packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -451,7 +422,7 @@ export class ObjectSetListenerWebsocket {
     payload: ObjectSetUpdates,
   ) => {
     const sub = this.#subscriptions.get(payload.id);
-    invariant(sub, `Expected subscription id ${payload.id}`);
+    if (sub == null) return;
```

Comment:
The old code assumed you would always be able to find by payload id. Is this no longer the case? Why?

Reply:
I forgot to test this, but at least in the new code when we delete the subscription in unsubscribe, without necessarily unsubscribing by sending a new request, this would error because we still received updates for that sub id. Even though we are unsubscribing and this should not happen, I wanted to remove this as a failsafe in case we still receive an update here

Reply:
Okay